CrowdStrikeFalcon/crowdstrike_falcon/event_stream_trigger.py

```diff
@@ -564,30 +560,8 @@ def client(self):
         )
 
     @cached_property
-    def verticles_collector(self) -> VerticlesCollector | None:
-        if (
-            self.configuration.tg_base_url is None
-            or self.configuration.tg_username is None
-            or self.configuration.tg_password is None
-        ):
-            self.log(
-                message="ThreatGraphAPI client disable: no credentials supplied",
-                level="info",
-            )
-            return None
-
-        try:
-            tg_client = CrowdstrikeThreatGraphClient(
-                self.configuration.tg_base_url,
-                self.configuration.tg_username,
-                self.configuration.tg_password,
-                default_headers=self._http_default_headers,
-            )
-
-            return VerticlesCollector(self, tg_client, self.client)
-        except Exception as error:
-            self.log_exception(error, message="Failed to start the verticles collector")
-            return None
+    def verticles_collector(self) -> VerticlesCollector:
+        return VerticlesCollector(self, self.client)
```

Comment:
What happens if the credentials don't have the ThreatGraph permission?
Can you provide a mechanism that disable the VerticlesCollector if the permission is missing (like [the one for AlertAPI](https://github.com/SEKOIA-IO/automation-library/blob/main/CrowdStrikeFalcon/crowdstrike_falcon/event_stream_trigger.py#L177-L189))?